travis-ci/managers/fuzzbuzz.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+set -x
+
+cd $REPO_ROOT
+wget https://app.fuzzbuzz.io/releases/cli/latest/linux/fuzzbuzz
+chmod +x fuzzbuzz
+./fuzzbuzz validate
+./fuzzbuzz target test fuzz-unit-file --all
```

Comment:
To judge from https://travis-ci.org/systemd/systemd/jobs/528541734, `fuzzbuzz` still exits with 0 even if something goes wrong. @everestmz could you take a look? It looks like at this point the FuzzBuzz CLI can't be used in scripts like the one I'm trying to add here to make sure that the fuzz targets are more or less OK.

Reply:
That should be fixed soon. In the meantime, I no longer run `fuzzbuzz` as root and according to  https://travis-ci.org/systemd/systemd/jobs/528544912 the script seems to be working.